Formula/asdf.rb

```diff
@@ -4,29 +4,17 @@ class Asdf < Formula
   url "https://github.com/asdf-vm/asdf/archive/v0.8.0.tar.gz"
   sha256 "9b667ca135c194f38d823c62cc0dc3dbe00d7a9f60caa0c06ecb3047944eadfa"
   license "MIT"
-  revision 1
+  revision 2
   head "https://github.com/asdf-vm/asdf.git"
 
   bottle :unneeded
 
-  depends_on "autoconf"
-  depends_on "automake"
-  depends_on "coreutils"
-  depends_on "libtool"
-  depends_on "libyaml"
-  depends_on "openssl@1.1"
-  depends_on "readline"
-  depends_on "unixodbc"
```

Comment:
It no longer depends on anything?

Reply:
@SMillerDev

If I were to make a calculated guess, it's because the expectation is that plugins such as `ruby` and `erlang` depends on those.

I only found 
```shell
get_concurrency() {
  if command -v nproc >/dev/null 2>&1; then
    nproc
  elif command -v sysctl >/dev/null 2>&1 && sysctl hw.ncpu >/dev/null 2>&1; then
    sysctl -n hw.ncpu
  elif [ -f /proc/cpuinfo ]; then
    grep -c processor /proc/cpuinfo
  else
    echo "1"
  fi
}
```

That depends on `coreutils`, until we can confirm that they're all expected, I suggest removing them. 

However, it's not used anywhere. I assume a plugin is supposedly going to use that, but in that case, the plugin itself should document the necessity, and not this formula. 

That's my take on it, I am more than happy to revert if someone asdf maintainer can confirm. 
Maybe @jthegedus can confirm if I am inaccurate in my guess.

Reply:
From https://asdf-vm.com/#/core-manage-asdf :
> **Dependencies**
> *Operating System*: macOS
> *Installation Method*: Homebrew
> `brew install coreutils curl git`

Unless the docs are in error, best to keep the abovementioned three dependencies in the formula.

Reply:
@gromgit I am not sure that's accurate.
For starters, macOS already comes with `curl` and homebrew curl` is keg-only, which means chances are it's not the `curl that asdf uses. 

> curl is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.


Which brings me to `coreutils`, again, I am not sure asdf uses the programs you think it is. 
`git` is the only one I get behind. 

I will investigate `coreutils` a bit further.

Reply:
@gromgit  Yeah doesn't seem like coreutils is necessary at all. 
Do you want to confirm this @jthegedus ?

Reply:
I am not sure I am the best to answer this. I have `coreutils` on my mac but have never tested `asdf` without it. Nor do I use Ruby or Homebrew.

At one point I naively added a large list of asdf-plugin common deps to the docs, but we culled those back to these three. I think `coreutils` was left out of caution. We have `coreutils` in our macOS test case. Running a PR with `coreutils` removed from the workflow would let you know.

Reply:
@jthegedus So I just tested without, and it works fine.
Added my forks of Ruby, Erlang and the origin Elixir plugins and installed all three. 
It's not necessary.

Reply:
So I just thought about something, it's probably worth it to add `coreutils` as a dependency since `asdf` maintainers shouldn't have to worry about differences from posix. I came to that when realizing it's supposed to be cross platform and divergent tools would cause issues. So not having to think about this, it makes sense to keep `coreutils` as a dependency even though at this moment, it makes no difference. So I added it again, together with `git` just for good measure.